### ISSUE TITLE
Support namespaces

### ISSUE THREAD

**Issue by kdambekalns** [iangreenleaf/Scisr#41]
When working with a namespaced codebase, the tool is of limited use. Here are some examples:

Renaming `Exception` to `RenamedException` changes too much (i.e. all `Exception` classes, but not all that's needed, so that `class Exception extends \TYPO3\Fluid\Core\Exception` becomes `class RenamedException extends \TYPO3\Fluid\Core\Exception`.

Using fully qualified class names matches only uses in docblocks, it seems. At least fully qualified use in code is left unchanged.

What should happen, is that renaming `Some\Namespaced\Class` to `Some\Namespaced\DifferentName` should just work. :)

**Comment by iangreenleaf** [comment]
Agree, this would be very nice. Unfortunately, I'm not putting a lot of time into Scisr development at this point. I'd definitely accept a pull request, though :)

**Comment by kdambekalns** [comment]
I whipped up some tests already and am currently looking into how Scisr actually works on the inside.
